mlir/include/mlir/Dialect/MIOpen/Tuning/GridwiseGemmParams.h

```diff
@@ -16,6 +16,8 @@
 #include "mlir/Dialect/MIOpen/MIOpenOps.h"
```

Comment:
There's too much content in this one file. Could you break it up into multiple files, also under a "solvers" directory, following MIOpen (further break down on conv_hip_implicit_gemm.cpp, and conv_hip_implicit_gemm_xdl.cpp)? A single set of performance config + solver stay in a single cpp file. You can have this one header file to include all the declarations.